bin/deploy

```diff
@@ -142,6 +142,19 @@ def deploy(client, cluster, service, repo, image, run_migrations = false)
   puts "#{service} deployment probably complete, but this tool is pretty rudimentary so check for yourself"
 end
 
+def notify_slack(env, image, stage)
+  slack_hook = ENV["SLACK_HOOK"]
+  if slack_hook
+    if stage=="start"
+      puts `curl -X POST --data-urlencode 'payload={"icon_emoji": ":ghost:", "text": "starting to deploy #{image} to #{env}"}' #{slack_hook}`
+    elsif stage=="end"
+      puts `curl -X POST --data-urlencode 'payload={"icon_emoji": ":sharkdance:", "text": "finished deploying #{image} to #{env}"}' #{slack_hook}`
+    end
+  end
```

Comment:
Put an else here where you print a message like "not notifying slack because blah, blah blah" and explain how to set it up

Reply:
ok done